### PR TITLE
Fix for using same name properties in binding.

### PR DESCRIPTION
**Pull request by nsndeck**
Fix for using same name properties in binding.

### CODE REVIEW

**Comment by nsndeck** [comment]
run ci

File: apps/tests/ui/bindable-tests.ts [hunk 1]
```diff
@@ -997,4 +997,73 @@ export function test_$ValueSupportWithinExpression() {
     model.set("anyColor", "red");
 
     TKUnit.assertEqual(bindableObj.get("test"), "red", "When anyColor is red test property should be red too.");
+}
+
+export function test_BindingToPropertiesWithSameNames() {
+    var model = new observable.Observable({
```
Comment (hshristov) [inline apps/tests/ui/bindable-tests.ts]: why don't expose item as field and skip the casts?

**Comment by nsndeck** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by enchev** [comment]
:+1:

**Pull request merged**